### PR TITLE
Homwork 10 is done. Create abstract factory

### PR DESCRIPTION
**Pull request by b-bodnar**
Has been created abstract factory which creates different transports

### CODE REVIEW

File: src/main/java/cursor/hw10/abstract_factory/execute/Executor.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package cursor.hw10.abstract_factory.execute;
+
+import cursor.hw10.abstract_factory.model.factory.TransportFactory;
+import cursor.hw10.abstract_factory.model.factory.impl.AmericanFactory;
+import cursor.hw10.abstract_factory.model.factory.impl.GermanFactory;
+import cursor.hw10.abstract_factory.model.transport.Car;
+import cursor.hw10.abstract_factory.model.transport.Plane;
+
```
Comment (omilovskiy) [inline src/main/java/cursor/hw10/abstract_factory/execute/Executor.java:8]: redundant line

File: src/main/java/cursor/hw10/abstract_factory/execute/Executor.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        Car car = factory.createCar();
+        car.go("New York");
+
+        Plane plane = factory.createPlane();
+        plane.fly("Perice");
+    }
+
+    private static void chooseFactory(int factoryID) {
+        switch (factoryID) {
+            case 1:
```
Comment (omilovskiy) [inline src/main/java/cursor/hw10/abstract_factory/execute/Executor.java]: what is 1, 2 ??  create a constants for this numbers

File: src/main/java/cursor/hw10/abstract_factory/model/transport/Car.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package cursor.hw10.abstract_factory.model.transport;
+
+public interface Car {
+
+    void go(String to);
```
Comment (omilovskiy) [inline src/main/java/cursor/hw10/abstract_factory/model/transport/Car.java]: maybe smth like: destinationPlace

File: src/main/java/cursor/hw10/abstract_factory/model/transport/Plane.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package cursor.hw10.abstract_factory.model.transport;
+
+public interface Plane {
+
+    void fly(String to);
```
Comment (omilovskiy) [inline src/main/java/cursor/hw10/abstract_factory/model/transport/Plane.java]: as previous

File: src/main/java/cursor/hw10/abstract_factory/execute/Executor.java (hunk 1 above)
Comment (b-bodnar) [reply src/main/java/cursor/hw10/abstract_factory/execute/Executor.java:8]: Done

File: src/main/java/cursor/hw10/abstract_factory/execute/Executor.java (hunk 2 above)
Comment (b-bodnar) [reply src/main/java/cursor/hw10/abstract_factory/execute/Executor.java]: Done

File: src/main/java/cursor/hw10/abstract_factory/model/transport/Car.java (hunk 3 above)
Comment (b-bodnar) [reply src/main/java/cursor/hw10/abstract_factory/model/transport/Car.java]: Done

File: src/main/java/cursor/hw10/abstract_factory/model/transport/Plane.java (hunk 4 above)
Comment (b-bodnar) [reply src/main/java/cursor/hw10/abstract_factory/model/transport/Plane.java]: Done

**Pull request merged**